packages/iTwinUI-react/src/core/Modal/Modal.tsx

```diff
@@ -177,52 +178,55 @@ export const Modal = (props: ModalProps) => {
 
   return !!container ? (
     ReactDOM.createPortal(
-      <CSSTransition
-        in={isOpen}
-        classNames='iui-dialog-animation'
-        timeout={{ exit: 600 }}
-        unmountOnExit={true}
-      >
+      <>
+        <div
+          className={cx('iui-backdrop', { 'iui-backdrop-visible': isOpen })}
+          tabIndex={-1}
+          onKeyDown={handleKeyDown}
+          ref={overlayRef}
+          onMouseDown={handleMouseDown}
+        />
         <FocusTrap>
-          <div
-            className={cx(
-              'iui-dialog-backdrop',
-              { 'iui-dialog-default': styleType === 'default' },
-              { 'iui-dialog-full-page': styleType === 'fullPage' },
-              { 'iui-dialog-visible': isOpen },
-              className,
-            )}
-            tabIndex={-1}
-            onKeyDown={handleKeyDown}
-            ref={overlayRef}
-            onMouseDown={handleMouseDown}
-            {...rest}
-          >
-            <div
-              className='iui-dialog'
-              id={id}
-              style={style}
-              role='dialog'
-              aria-modal='true'
+          <div>
+            <CSSTransition
+              in={isOpen}
+              classNames='iui-dialog-animation'
+              timeout={{ exit: 600 }}
+              unmountOnExit={true}
             >
-              <div className='iui-dialog-title-bar'>
-                <div className='iui-dialog-title'>{title}</div>
-                {isDismissible && (
-                  <IconButton
-                    size='small'
-                    styleType='borderless'
-                    onClick={onClose}
-                    aria-label='Close'
-                  >
-                    <SvgClose />
-                  </IconButton>
+              <div
+                className={cx(
+                  'iui-dialog',
+                  { 'iui-dialog-default': styleType === 'default' },
+                  { 'iui-dialog-full-page': styleType === 'fullPage' },
+                  { 'iui-dialog-visible': isOpen },
+                  className,
                 )}
+                id={id}
+                style={style}
+                role='dialog'
+                aria-modal='true'
+                {...rest}
```

Comment:
style and className were previously applied on the backdrop/wrapper whereas now they will be on the dialog itself. This makes sense but we should mention this in the changelog as it's is a breaking change for user CSS.